### ISSUE TITLE
Re-running XCUITests after failure consistently fails with "App accessibility isn't loaded"

### ISSUE THREAD

**Issue by AaronSofaer** [bitrise-io/steps-xcode-test#70]
Stack: XCode 8.2.x

When XCUITests are run with the option to re-run them on failure selected, the re-run tests all hang for one minute and then fail with the error message "App accessibility isn't loaded".

Output log or a link to an example can be provided if desired.

**Comment by viktorbenei** [comment]
Thanks for reporting! We'll check if we can improve this, but this might very well be an Xcode 8.2 (beta!) issue

**Comment by AaronSofaer** [comment]
One of the stack overflow comments suggested that resetting the container between tests might resolve the issue; supposedly this line of code will do the trick, but I haven't tried it myself.

`killall "Simulator" 2> /dev/null; xcrun simctl erase all`

You'd have to then allow enough time for the simulator to spin back up, of course.

**Comment by viktorbenei** [comment]
Thanks for the info, we'll see what we can do!

Do you have a project where this issue can be reproduced?

**Comment by AaronSofaer** [comment]
Not consistently, now that my tests are passing on the first run. This is a failure:  https://www.bitrise.io/build/a5ddf035bd012a6d 

I can create a branch that consistently fails if that would be of assistance... I don't know if the complexity of our app / code has an effect on the manifestation of the bug.

**Comment by viktorbenei** [comment]
@AaronSofaer we added quite a few workarounds/retries to the step since you opened this issue - should we keep this open? does this still happen?

(for now I'll close the issue, but if you reply that this should be reopened I'll reopen it right away)
